frontend/lib/src/components/elements/Json/Json.test.tsx

```diff
@@ -54,31 +57,23 @@ describe("JSON element", () => {
       "numbers":[ -1e27, NaN, Infinity, -Infinity, 2.2822022,-2.2702775],
     }`,
     })
-    const wrapper = mount(<Json {...props} />)
-    expect(wrapper).toBeDefined()
+    render(<Json {...props} />)
+    expect(screen.getByTestId("stJson")).toBeInTheDocument()
   })
 
-  it("picks a reasonable theme when the background is light", () => {
-    // <Json> uses `hasLightBackgroundColor` to test whether our theme
-    // is "light" or "dark". Mock the return value for the test.
-    jest.spyOn(themeUtils, "hasLightBackgroundColor").mockReturnValue(true)
-
-    const props = getProps()
-    const wrapper = mount(<Json {...props} />)
-
-    expect(wrapper.find('[theme="rjv-default"]').exists()).toBeTruthy()
-    expect(wrapper.find('[theme="monokai"]').exists()).toBeFalsy()
-  })
-
-  it("picks a reasonable theme when the background is dark", () => {
-    // <Json> uses `hasLightBackgroundColor` to test whether our theme
-    // is "light" or "dark". Mock the return value for the test.
-    jest.spyOn(themeUtils, "hasLightBackgroundColor").mockReturnValue(false)
-
-    const props = getProps()
-    const wrapper = mount(<Json {...props} />)
+  describe("getJsonTheme", () => {
+    it("picks a reasonable theme when the background is light", () => {
```

Comment:
I know this was kinda how the old test was written, but don't love testing which theme passed this way - could we instead do something like:
```javascript
jest.spyOn(themeUtils, "hasLightBackgroundColor").mockReturnValue(true)

render(<Json {...getProps()} />)
// checks resulting json coloration based on theme passed
expect(screen.getByText("}")).toHaveStyle("color: rgb(0, 43, 54)")
```